lib/mito/utilities/SharedPointer.h

```diff
@@ -32,6 +32,9 @@ namespace mito::utilities {
         // operator->
         auto operator->() const -> handle_t;
 
+        // // operator*
+        // auto operator*() const -> const resource_t &;
```

Comment:
remove

Reply:
I'd rather leave it as we may want to implement {operator*} for shared pointer in the future.